src/zimCheck.h

```diff
@@ -85,8 +85,9 @@ void getDependencies(std::string page, std::vector <std::string> *links)
                 {
                     i++;
                 }
-                length=i-startingPoint;
-                links->push_back(page.substr(startingPoint,length));
+                std::string url = page.substr(startingPoint, i-startingPoint);
+                if (std::regex_match(url, std::regex("http(s|)://")))
```

Comment:
And `std::regex` should be created only once as a "pseudo global" variable instead of being created for each url's test.

Reply:
I made it static, hope this is what you meant.